utils/i18n/strings.ts

```diff
@@ -41,6 +41,7 @@ export const strings = {
   are_you_sure: "Are you sure?",
   promote_to_admin: "Promote to admin",
   send_a_message: "Send a message",
+  client_error: "Looks like something went wrong, please try reconnecting.",
```

Comment:
If I'm reading useShouldShowErrored correctly, this is actually a network state indicator rather than a generic error state. I think either the copy and presentation should reflect that, or, the error should react more directly to a client error rather than whether the internet is reachable.

![IMG_4678](https://github.com/user-attachments/assets/33148917-d528-4e3a-abb6-054899a232a9)
![IMG_4679](https://github.com/user-attachments/assets/311ef5c6-3550-426f-9616-288c523a8d15)

Reply:
@saulmc _should_ be the opposite https://github.com/ephemeraHQ/converse-app/pull/424/files#diff-0019535d4794401d7721f32799550bf423ef51d9630bfe9e4feb6fd2050b3200R8
Should only show if you are connected to the internet and the client creation failed

Reply:
Misread! Carry on. Still think it should replace the spinner since "never going to load" and "taking really long to load" are mutually exclusive.